knowledge-graph/src/main/scala/io/renku/knowledgegraph/docs/Endpoint.scala

```diff
@@ -45,30 +46,34 @@ private class EndpointImpl[F[_]: Async](serviceVersion: ServiceVersion) extends
            "Get info about datasets, users, activities, and other entities".some,
            serviceVersion.value
       )
-    ).addServer(renkuLabServer)
-      .addPath(lineage.EndpointDoc.path)
-      .addSecurity(securityScheme)
+    ).addPath(lineage.EndpointDoc.path)
+      .addServer(server)
+      .addSecurity(privateToken)
+      .addSecurity(oAuth)
+      .addNoAuthSecurity
       .addResponsesToAll(authErrorResponses)
 
-  private lazy val localServer =
-    Server("http://localhost:{port}/{basePath}",
-           "Local server",
-           Map("port" -> Variable("8080"), "basePath" -> Variable("knowledge-graph/projects"))
+  private lazy val server: Server =
+    Server(
+      url = "knowledge-graph",
```

Comment:
Do you think `knowledge-graph` would do or we'd need to have `/knowledge-graph`?